GOVERNANCE.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Governance
+layout: page
+
+---
```

Comment:
what's this header for?

Reply:
I copied it over from the source, it has to do with css I believe, which we don't have, so I'll remove it.